packages/workspace/src/schematics/remove/lib/update-tsconfig.ts

```diff
@@ -23,7 +23,12 @@ export function updateTsconfig(schema: Schema) {
 
         const tsConfigPath = 'tsconfig.json';
         if (tree.exists(tsConfigPath)) {
-          let contents = JSON.parse(tree.read(tsConfigPath).toString('utf-8'));
+          let contents = tree.read(tsConfigPath).toString('utf-8');
```

Comment:
Could you instead readJsonInTree? It throws a good error and also strips comments

Reply:
Sure!!, changes were applied 👍 

The variable was renamed to keep the same style as
`const nxJson = readJsonInTree<NxJson>(tree, 'nx.json');`